webpack.config.js

```diff
@@ -24,15 +35,30 @@ config.module.loaders = [
     query: {
       presets: ['es2015', 'react']
     }
+  },
+  {
+    test: /\.styl$/,
+    loader: ExtractTextPlugin.extract('css-loader?modules&camelCase&localIdentName=[name]__[local]___[hash:base64:5]!stylus-loader') // eslint-disable-line
   }
 ];
 
 config.plugins = [
   new webpack.DefinePlugin({
     'process.env': {
-      'NODE_ENV': JSON.stringify('production')
+      NODE_ENV: JSON.stringify('development')
```

Comment:
Probably not, but I'll check this when the focus is on project compilation and automation(another issue).